config.yml

```diff
@@ -1934,6 +1945,11 @@ nodes:
         type: location?
       - name: parts
         type: node[]
+        kind:
+          - StringNode
+          - EmbeddedStatementsNode
+          - EmbeddedVariableNode
+          - InterpolatedStringNode # example in parse_line_evstr_after_break.txt
```

Comment:
This one is by design. It represents string concatenation (we replaced `StringConcatNode` with this).